Content.Client/Disposal/Systems/DisposalUnitSystem.cs

```diff
@@ -10,18 +10,18 @@ namespace Content.Client.Disposal.Systems
 {
     public sealed class DisposalUnitSystem : SharedDisposalUnitSystem
     {
-        public List<DisposalUnitComponent> PressuringDisposals = new();
+        public List<EntityUid> PressuringDisposals = new();
```

Comment:
can you make this static?

Reply:
Made it `private` instead; no need for any other systems to be using this field directly.

Reply:
is there a reason to not make it static? is more of a convention to REALLY make sure no new instances are made since this is a system and not a component

Reply:
Won't making it static have problems with multithreading, specifically stuff like tests?